### PR TITLE
add multiprocessing tasks

### PR DESCRIPTION
**Pull request by valegeri**
add some files closely related to multitasking

### CODE REVIEW

File: multi.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    def __call__(self):
+        time.sleep(0.1)
+        return self.a, self.b
+
+    def __str__(self):
+        return f'{self.a} {self.b}'
+
+
+if __name__ == '__main__':
+    tasks_queue = JoinableQueue()
```
Comment (weratt) [inline multi.py:42]: Dont get it

Comment (valegeri) [reply multi.py:42]: multitasking

**Review by weratt — CHANGES_REQUESTED** [review]
Add newlines

File: __init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+import timer
```
Comment (weratt) [inline __init__.py]: Add newline

File: multi.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        tasks_queue.put(None)
+
+    tasks_queue.join()
+
+    for w in workers:
+        w.join()
+
+    for i in range(num_proc):
+        result = results_queue.get()
+        print(f'Result: {num_proc}:{result}')
```
Comment (weratt) [inline multi.py]: Add newline

**Review by weratt — CHANGES_REQUESTED** [review]
Add newlines

File: __init__.py (hunk 2 above)
Comment (valegeri) [reply __init__.py]: Done

File: multi.py (hunk 3 above)
Comment (valegeri) [reply multi.py]: Done

**Review by weratt — APPROVED** [review]

**Pull request merged**